editor/editor_resource_picker.cpp

```diff
@@ -327,6 +327,13 @@ void EditorResourcePicker::_edit_menu_cbk(int p_which) {
 
 		case OBJ_MENU_PASTE: {
 			edited_resource = EditorSettings::get_singleton()->get_resource_clipboard();
+			if (Engine::get_singleton()->is_editor_hint() && edited_resource->is_built_in() && EditorNode::get_singleton()->get_edited_scene() &&
```

Comment:
This is an editor node, is there any case where `is_editor_hint` wouldn't be true?